types/mongodb/index.d.ts

```diff
@@ -11,10 +11,11 @@
 //                 Julien Chaumond <https://github.com/julien-c>
 //                 Dan Aprahamian <https://github.com/daprahamian>
 //                 Denys Bushulyak <https://github.com/denys-bushulyak>
+//                 Bastien Arata <https://
```

Comment:
This looks wrong. Shouldn't this be a valid url and close the tag?

Reply:
Yep thanks, forgot to paste my account link